amy/communityroles/migrations/0001_initial.py

```diff
@@ -57,12 +56,13 @@ class Migration(migrations.Migration):
                 ('start', models.DateField(blank=True, null=True)),
                 ('end', models.DateField(blank=True, null=True)),
                 ('url', models.URLField(blank=True, default='', verbose_name='URL')),
-                ('generic_relation_m2m', django.contrib.postgres.fields.ArrayField(base_field=models.PositiveIntegerField(), default=list, size=None)),
-                ('award', models.ForeignKey(null=True, on_delete=django.db.models.deletion.PROTECT, to='workshops.Award')),
+                ('award', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.PROTECT, to='workshops.Award')),
                 ('config', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='communityroles.CommunityRoleConfig')),
-                ('inactivation', models.ForeignKey(null=True, on_delete=django.db.models.deletion.PROTECT, to='communityroles.CommunityRoleInactivation')),
-                ('membership', models.ForeignKey(null=True, on_delete=django.db.models.deletion.PROTECT, to='workshops.Membership')),
+                ('inactivation', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.PROTECT, to='communityroles.CommunityRoleInactivation')),
+                ('membership', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.PROTECT, to='workshops.Membership')),
                 ('person', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+                ('generic_relation_content_type', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to='contenttypes.ContentType')),
+                ('generic_relation_pk', models.PositiveIntegerField(blank=True, null=True)),
```

Comment:
Since we're still in development phase of this project, I'm going to allow myself to modify this migration.